packages/apify/src/actor.ts

```diff
@@ -1580,32 +1585,100 @@ export interface ApifyEnv {
     token: string | null;
 
     /**
-     * Date when the actor was started (ACTOR_STARTED_AT)
+     * Date when the actor was started (APIFY_STARTED_AT)
      */
     startedAt: Date | null;
 
     /**
-     * Date when the actor will time out (ACTOR_TIMEOUT_AT)
+     * Date when the actor will time out (APIFY_TIMEOUT_AT)
      */
     timeoutAt: Date | null;
 
     /**
      * ID of the key-value store where input and output data of this
-     * actor is stored (ACTOR_DEFAULT_KEY_VALUE_STORE_ID)
+     * actor is stored (APIFY_DEFAULT_KEY_VALUE_STORE_ID)
      */
     defaultKeyValueStoreId: string | null;
 
     /**
      * ID of the dataset where input and output data of this
-     * actor is stored (ACTOR_DEFAULT_DATASET_ID)
+     * actor is stored (APIFY_DEFAULT_DATASET_ID)
      */
     defaultDatasetId: string | null;
 
     /**
      * Amount of memory allocated for the actor,
-     * in megabytes (ACTOR_MEMORY_MBYTES)
+     * in megabytes (APIFY_MEMORY_MBYTES)
      */
     memoryMbytes: number | null;
+
+    /**
+     * If set to 1, the web browsers inside the Actor should run in headless
+     * mode because there is no windowing system available. (APIFY_HEADLESS)
+     */
+    headless: 1 | null;
```

Comment:
Took a look at Lukas' snapshot and it's string "1" there even. I guess that's because this funciton only cares about integer conversion and base on the following list, and it's missing there

```
export declare const INTEGER_ENV_VARS: readonly ["ACTOR_MAX_PAID_DATASET_ITEMS", "ACTOR_MEMORY_MBYTES", "ACTOR_WEB_SERVER_PORT", "ACTOR_MAX_PAID_DATASET_ITEMS", "APIFY_CONTAINER_PORT", "APIFY_DEDICATED_CPUS", "APIFY_MEMORY_MBYTES", "APIFY_METAMORPH_AFTER_SLEEP_MILLIS", "APIFY_PERSIST_STATE_INTERVAL_MILLIS", "APIFY_PROXY_PORT", "APIFY_SYSTEM_INFO_INTERVAL_MILLIS"];
```

Reply:
regarding it being here, the return type is the mirror of APIFY_ENV_VARS which lists it. And there is no actor equivalent

Reply:
Sorry, I mistakenly created new review. I wanted to suggest using booleans but now I realized this is only updating the types. And changing the values would be breaking change even though they should have been parsed as booleans.